src/gui/owncloudgui.cpp

```diff
@@ -347,6 +348,15 @@ void ownCloudGui::addAccountContextMenu(AccountStatePtr accountState, QMenu *men
     actionOpenoC->setProperty(propertyAccountC, QVariant::fromValue(accountState->account()));
     QObject::connect(actionOpenoC, SIGNAL(triggered(bool)), SLOT(slotOpenOwnCloud()));
 
+    if (accountState->state() == AccountState::Connected) {
+        menu->addSeparator();
+
+        QString quotaInfo = tr("Fetching Quota");
+        auto actionQuotaInfo = menu->addAction(quotaInfo);
+        actionQuotaInfo->setEnabled(false);
+        new QuotaAction(actionQuotaInfo, accountState);
```

Comment:
Ah seems we can drop the OS X magic when 5.6.2 is used: https://github.com/owncloud/client/issues/5470 https://bugreports.qt.io/browse/QTBUG-54845